js/pages/prediction/components/PopulationSettingsEditor.html

```diff
@@ -0,0 +1,68 @@
+<div class="paddedWrapper">
+    <div>
+        <label>Define the time-at-risk window start, relative to target cohort entry:</label>
+        <div class="input-group">
+            <span contenteditable="true" class="numericInputField dropdown" data-bind="htmlValue: $component.populationSettings.riskWindowStart, eventType:'blur', ko_autocomplete: { value: $component.populationSettings.timeAtRiskStart, source: $component.options.dayOptions, minLength: 0, maxShowItems: 10, scroll: true }" />&nbsp;days from cohort start date
+        </div>
+    </div>
+</div>
+<div>
+    <label>Add exposure days to start?</label>
+    <div class="input-group">
+        <select class="form-control" data-bind="options: $component.options.yesNoOptions, optionsText:'name', optionsValue:'id', value: $component.populationSettings.addExposureDaysToStart"></select>
+    </div>
+</div>
+<div>
+    <label>Define the time-at-risk window end:</label>
+    <div class="input-group">
+        <span contenteditable="true" class="numericInputField dropdown" data-bind="htmlValue: $component.populationSettings.riskWindowEnd, eventType:'blur', ko_autocomplete: { value: $component.populationSettings.riskWindowEnd, source: $component.options.dayOptions, minLength: 0, maxShowItems: 10, scroll: true }" />&nbsp;days from cohort start date
```

Comment:
Usage of `span` for what is input in fact, is a bit...hm..not very logical as for me and spoils sematics. Pls get a look how it is properly done in Cohort Pathways

Reply:
The reason for the us of span is that it can grow/shrink based on the content of the DOM.  Perhaps there has been recent advances in CSS and DOM technology, but this is why the people on the HTML specification committee added the 'contenteditable' attribute to the element.

And as far as 'proper', please please do not use inflammatory language when suggesting an alternative.  The Pathways implementation desired the static with in the input fields because of their arrangement.  If we did it 'properly' in cohort definition editor here:
![image](https://user-images.githubusercontent.com/6818777/46548316-bc099c80-c89c-11e8-8bb3-0d52f781ebcf.png)

vs if we went with pathways:
![image](https://user-images.githubusercontent.com/6818777/46548731-e3ad3480-c89d-11e8-88d9-59196d1d0240.png)

Which  feel the compressed input fields makes sense here for cohort definitions. Does it make sense in PLE? Why not ask the question instead of passing judgement?

Reply:
Why I named it not proper? Because I can fall into such situation:
![2018-10-05 15_11_42-_atlas_ cohort definitions unsaved](https://user-images.githubusercontent.com/10010071/46555482-786d5d80-c8b1-11e8-99db-1c44a2ce0c0d.png)

Why do I receive this? Because `contenteditable` is rich-text editor, which is not what required for a single digit. Your screenshot is a bit unfair: although you are right about fixed width, there is no need to do such wide fields. So smth like 5-digit wide should look OK.

Reply:
>  Your screenshot is a bit unfair:

It's just an input box, what your screenshot is showing is that the user is free to type in whatever they want into it. What your screen shot is not showing is that the numeric validation rejects that as an input and everything snaps back down to size.  Unfair-says-what?

As far as my example being unfair,  took the exact dimensions from the TXP pathways UI to capture the _exact_ same range of numeric inputs for those parameters.  I don't know how i could have been more fair.

We're getting off track here, the commentary was just how your feedback is being delivered:  making statements about what is 'proper' without understanding what the functionality requirements are.  So, I suggest we answer the question 'What is the desired behavior here' first, next ask 'Is the approach X applicable for this situation', and then no one feels like their efforts are degraded.